.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+    "semi": true,
+    "tabWidth": 2,
+    "printWidth": 100,
+    "singleQuote": true,
+    "trailingComma": "none",
```

Comment:
Think we should allow trailing commas since they're widely used among at least 3 of OP's projects.